test/contracts/promises/three-contracts/first-contract/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+#![no_std]
+#![no_main]
```

Comment:
No need for no_main crates of type `cdylib`. You can safely remove.

Reply:
Removed.